tests/cnf/ran/internal/ranparam/const.go

```diff
@@ -27,6 +27,10 @@ const (
 	PtpDaemonsetLabelSelector = "app=linuxptp-daemon"
 	// LogLevel is the verbosity for ran/internal packages.
 	LogLevel glog.Level = 80
+	// OpenshiftLoggingNamespace is the namespace where logging pods are.
+	OpenshiftLoggingNamespace = "openshift-logging"
+	// MasterNodeSelector Master node selector.
+	MasterNodeSelector = "node-role.kubernetes.io/master="
```

Comment:
node-role master is deprecated in favor of control-plane, depending on which version you need to support it may not be necessary to include. The control plane selector is provided in GeneralConfig

Reply:
fixed